server/tests/acc-users.test.js

```diff
@@ -1,347 +1,347 @@
-import chai from 'chai';
-import chaiHttp from 'chai-http';
-import jwt from 'jsonwebtoken';
-import dotenv from 'dotenv';
-import server from '../app.js'
+// import chai from 'chai';
+// import chaiHttp from 'chai-http';
+// import jwt from 'jsonwebtoken';
+// import dotenv from 'dotenv';
```

Comment:
You have commented codes

Reply:
Now I delete those files and commented codes. You may recheck review it again!